packages/terra-time-input/tests/jest/TimeInput.test.jsx

```diff
@@ -96,6 +96,23 @@ it('should render a 12 hour timepicker meridiem with buttons when viewed on a mo
   delete window.ontouchstart;
 });
 
+it('should not have duplicate ids on the page when multiple date pickers are initialized', () => {
```

Comment:
Since the consumer must supply a unique `name` to receive a unique `id` is this test needed?

Reply:
Yeah. If I left the names as time-input, time-input-2, time-input-3, it would generate three time-inputs all with the same id's of terra-time-hour-time-input and terra-time-minute-time-input. I want to ensure this test generates terra-time-hour-time-input-1,  terra-time-hour-time-input-2, and  terra-time-hour-time-input-3.